.travis.yml

```diff
@@ -34,6 +34,13 @@ jobs:
         - docker run -d --name bblfshd --privileged -v $HOME/bblfshd:/var/lib/bblfshd -p "9432:9432" bblfsh/bblfshd
         - docker exec -it bblfshd bblfshctl driver install go bblfsh/go-driver:v2.0.3
         - make test-sdk
+    - name: "SDK Integration Tests macOS"
+      if: NOT type = pull_request
```

Comment:
Could you please also help to understand, why we need this _only_ for macOS?

Reply:
The issue description says to run these new tests only on master.
I remember we discussed doing this to avoid making the tests for PRs take longer, while still running everything that we merge to master on macOS.

Reply:
yep. Sometimes travis starts macos container with a big delay.
It's enough to run them only on master. In case something went wrong we will receive an email.

Reply:
👍 thank you!

Reply:
Shall we also document `make test-sdk-short` in https://github.com/src-d/lookout/blob/master/CONTRIBUTING.md#testing ?